spiketools/tests/plts/test_annotate.py

```diff
@@ -6,8 +6,9 @@
 from spiketools.tests.tutils import plot_test
 
 from spiketools.plts.annotate import *
```

Comment:
I think we can remove this line?

Reply:
line 8? You're right that since most of the functions are 'private' (have a leading underscore and are not included in `import *`, the `color_pval` is imported in this line. Also, as a convention, I would tend to leave it in that each test file imports everything from each 'to be tested' file